bridge/client/app/_components/ktb-subscription-item/ktb-subscription-item.component.scss

```diff
@@ -1,7 +1,15 @@
+@import '~@dynatrace/barista-components/core/src/style/variables';
+
 .dt-icon {
   width: 20px;
 }
 
 .item {
   height: 30px;
 }
+
+button.disabled {
```

Comment:
There should already be a style for `button[disabled]` so if you use the attribute instead of a class there should be no need for defining new styles.

Reply:
I just used this for a disabled button that is not really disabled (because the overlay would not work). I already changed the implementation and removed this style definition